packages/lib/src/components/internal/PhoneInput/PhoneInput.tsx

```diff
@@ -61,8 +61,6 @@ export function PhoneInput(props) {
                                 />
 
                                 <div className="adyen-checkout__phoneNumber">
-                                    <div>{data.phonePrefix}</div>
```

Comment:
The reasons to remove this line are:
- We already show the phone prefix as the selected label due to the changes in `packages/lib/src/components/QiwiWallet/utils.ts`, showing another `phonePrefix` is a duplication and doesn't look good.
- `packages/lib/src/components/internal/PhoneInput/PhoneInput.tsx` is only used by qiwi wallet, so it's safe to remove the line as well